lib/routes/index.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as vc from '@digitalbazaar/vc';
+import assert from 'assert-plus';
+import {
+  createDiscloseCryptosuite as createEcdsaSd2023DiscloseCryptosuite
+} from '@digitalbazaar/ecdsa-sd-2023-cryptosuite';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import express from 'express';
+import {loader} from '../documentLoader.js';
+
+const documentLoader = loader.build();
+
+export const router = express.Router();
+
+router.post('/derive', asyncHandler(async (req, res) => {
+  const {selectivePointers, verifiableCredential} = req.body;
```

Comment:
A goal here is to make this endpoint be as VC API compliant as possible -- I suggest we nest `selectivePointers` under `options`:

```suggestion
  const {options: {selectivePointers}, verifiableCredential} = req.body;
```

Reply:
Updated [79e0e59](https://github.com/digitalbazaar/vc-holder-http/pull/1/commits/79e0e5965924fac0ff54875a75230eb6f2af505a)